components/tooltip/demo/tooltip.html

```diff
@@ -163,7 +156,7 @@ <h2>Help Tooltip</h2>
 		<d2l-demo-snippet>
 			<template>
 				<d2l-tooltip-help text="Helpful label">Contents should elaborate on the label (be short and concise)</d2l-tooltip-help>
-				<p class="d2l-body-small">
+				<p class="d2l-body-small" style="padding-top: 20px;">
```

Comment:
Yep, this change is unrelated but I just tacked it onto this PR anyway while I was at it.

I added a new vdiff test, since then we could capture the "margins being removed correctly" in automation. Or are you suggesting to also add that to the demo page?

Reply:
Yeah I was saying demoing paragraphs nested in the tooltip, if that's a supported use case.

Reply:
I think it's more of an edge case, to be honest.

I believe the biggest use case for a tooltip is just passing in plain text or so. 
However, it could be possible (since we use slotting) that a consumer passes in a paragraph, and this was an attempt to improve that a bit since we don't desire the margins that paragraphs come with by default.

To me, it feels a bit weird to create a demo for it in the demo page, since it's less of a distinct "thing" compared to what we showcase in the other demos, I suppose. 

If someone on our team wanted to double-check that, they should be able to just generate the vdiff golden. 
Though I suppose for ease of use a demo in that case would make sense.